client/state/reader/follows/test/reducer.js

```diff
@@ -416,4 +420,125 @@ describe( 'reducer', () => {
 			expect( state ).to.equal( original );
 		} );
 	} );
+
+	describe( 'follow', () => {
+		it( 'should mark an existing feed as followed and leave the rest alone', () => {
+			const original = deepFreeze( {
+				'example.com': {
+					is_following: false,
+					blog_ID: 123
+				}
+			} );
+
+			const state = items( original, follow( 'http://example.com' ) );
+			expect( state ).to.eql( {
+				'example.com': {
+					is_following: true,
+					feed_URL: 'http://example.com',
+					blog_ID: 123
+				}
+			} );
+		} );
+
+		it( 'should create a new entry for a new follow', () => {
+			const state = items( {}, follow( 'http://example.com' ) );
+			expect( state ).to.eql( {
+				'example.com': {
+					feed_URL: 'http://example.com',
+					is_following: true
+				}
+			} );
+		} );
+	} );
+
+	describe( 'unfollow', () => {
+		it( 'should mark an existing follow as unfollowed', () => {
+			const original = deepFreeze( {
+				'example.com': {
+					is_following: true,
+					feed_URL: 'http://example.com',
+					blog_ID: 123
+				}
+			} );
+
+			const state = items( original, unfollow( 'http://example.com' ) );
+			expect( state ).to.eql( {
+				'example.com': {
+					is_following: false,
+					feed_URL: 'http://example.com',
+					blog_ID: 123
+				}
+			} );
+		} );
+
+		it( 'should return the original state when already unfollowed', () => {
+			const original = deepFreeze( {
+				'example.com': {
+					is_following: false,
+					feed_URL: 'http://example.com',
+					blog_ID: 123
+				}
+			} );
+
+			const state = items( original, unfollow( 'http://example.com' ) );
+			expect( state ).to.equal( original );
```

Comment:
`.to.equal` is actually `===`, not deep. http://chaijs.com/api/bdd/#method_equal